packages/demo/webpack.config.js

```diff
@@ -0,0 +1,78 @@
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const path = require('path');
+const isTravis = !!process.env.TRAVIS;
+
+/**
+ * Config file for the documentation project
+ */
+module.exports = {
+    entry: {
+        main: './src/Index.tsx',
+    },
+    mode: isTravis ? 'production' : 'development',
+    output: {
+        path: path.join(__dirname, '/dist'),
+        filename: '[name].bundle.js',
+        chunkFilename: 'assets/[name].bundle.js',
+    },
+    devtool: isTravis ? 'source-map' : 'eval-source-map',
+    resolve: {
+        extensions: ['.ts', '.tsx', '.js', '.jsx'],
+    },
+    plugins: [
+        new HtmlWebpackPlugin({
+            title: 'Vapor Design System',
+            favicon: 'src/favicon.ico',
+            chunks: ['main'],
+            template: 'src/index.html',
+        }),
+        new webpack.HotModuleReplacementPlugin(),
+        new webpack.ContextReplacementPlugin(/moment[/\\]locale$/, /en-ca/),
+    ],
+    stats: 'minimal',
+    module: {
+        rules: [
+            {
+                test: /\.tsx?$/,
+                use: [
+                    {
+                        loader: 'ts-loader',
+                        options: {
+                            projectReferences: true,

```

Comment:
You should use `transpileOnly` with the fork-ts-checker since we don't need the d.ts to be generated

You should also add `experimentalWatchApi: true,`

Reply:
experimentalWatchApi is there by default now https://github.com/TypeStrong/ts-loader/issues/757